vcd/resource_vcd_vapp.go

```diff
@@ -249,6 +255,26 @@ func resourceVcdVAppCreate(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	if _, ok := d.GetOk("properties"); ok {
+		// TODO - this is just for testing. Real issue must be solved
+		time.Sleep(10 * time.Second)
+		vapp, err := vdc.FindVAppByName(d.Get("name").(string))
+		if err != nil {
+			return fmt.Errorf("unable to find vApp by name %s: %s", d.Get("name").(string), err)
+		}
+
+		vappProperties, err := getProductSectionListType(d)
```

Comment:
I think here we operate with State file, in state file we have guest properties.

Reply:
changed name